internal/oci/stats.go

```diff
@@ -24,24 +28,37 @@ type ContainerStats struct {
 }
 
 // Returns the total number of bytes transmitted and received for the given container stats
-func getContainerNetIO(stats *libcontainer.Stats) (received, transmitted uint64) {
-	for _, iface := range stats.Interfaces {
-		received += iface.RxBytes
-		transmitted += iface.TxBytes
-	}
-	return
+func getContainerNetIO(netNsPath string) (received, transmitted uint64) {
+	ns.WithNetNSPath(netNsPath, func(_ ns.NetNS) error { // nolint: errcheck
+		link, err := netlink.LinkByName(ocicni.DefaultInterfaceName)
```

Comment:
are we sure we used ocicni.DefaultInterfaceName?

Reply:
I thought so. Do we have other Interfaces inside containers than eth0?

[I mean the logic is now a bit turned around since we have now the containers perspective.]